packages/nextjs/src/config/loaders/dataFetchersLoader.ts

```diff
@@ -136,13 +155,26 @@ export default function wrapDataFetchersLoader(this: LoaderThis<LoaderOptions>,
     if (hasDefaultExport(ast)) {
       outputFileContent += `
         import { default as _sentry_default } from "${this.resourcePath}?sentry-proxy-loader";
-        import { withSentryGetInitialProps } from "@sentry/nextjs";
-
-        if (typeof _sentry_default.getInitialProps === 'function') {
-          _sentry_default.getInitialProps = withSentryGetInitialProps(_sentry_default.getInitialProps);
-        }
-
-        export default _sentry_default;`;
+        import { withSentryGetInitialProps } from "@sentry/nextjs";`;
+
+      if (this.resourcePath.match(underscoreAppRegex)) {
```

Comment:
If you already have the parameterized route, why do you need these regexes? Can't you just test the route against the strings `'_app'`, `'_document'`, and `'_error'`, since we know those have to live at the top level of `pages`?

Reply:
Excellent suggestion - thanks! Removed in ae9cb73f2817a693bc0a08324f2539460b882ccf